camelot/core/files/storage.py

```diff
@@ -133,7 +133,7 @@ def _create_tempfile_with_user_exceptions(self, suffix: str, prefix: str) -> Tup
     def _create_tempfile(self, suffix: str, prefix: str) -> Tuple[int, str]:
         return tempfile.mkstemp(suffix=suffix, prefix=prefix, dir=self.upload_to)
 
-    def checkin(self, local_path: PurePosixPath, filename: os.PathLike = None) -> StoredFile:
+    def checkin(self, local_path: PurePath, filename: os.PathLike = None) -> StoredFile:
```

Comment:
ik zou denken dat checkin een 'Path' moet zijn ipv een 'PurePath'

Reply:
Eigenlijk niet want we gaan de copy functie van shutil gebruiken en die aanvaard gewoon path objecten, we gaan niets met het path object zelf doen. kwestie van een bestaand pad af te dwingen gaat het niet helpen om een gewoon Path te gebruiken

Reply:
wat de functie doet is eigenlijk nt zo van belang, het is eerder de interface definitie vd functie, die dan aangeeft dat het path is dat 'bestaat on disk'